test/greet-test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const assert = require('assert');
+
+describe('Greet Module', function() {
+  describe('#greet', function() {
+    it('should return hello Geoff', function() {
+      assert.ok(greet.greet('Geoff') === 'hello Geoff', 'not equal to hello Geoff');
+    });
+    it('should throw error for missing name', function() {
+      assert.throws(function() {
+        greet.greet();
+      });
+    });
+  });
+});
```

Comment:
Good test implementation.

As a sidenote (in case you were wondering how to test the bonus) I'm sure you probably discovered that mocha doesn't accept command line arguments so testing for process.argv[2] will always return undefined. In which case you could use a work around and check for a truthy value for the test, although that's rather hacky and there's a much better way to go about it which we will get to later this week!